hack/istio/multicluster/install-multi-primary.sh

```diff
@@ -211,11 +235,19 @@ source ${SCRIPT_DIR}/setup-ca.sh
 
 echo "==== INSTALL ISTIO ON CLUSTER #1 [${CLUSTER1_NAME}] - ${CLUSTER1_CONTEXT}"
 switch_cluster "${CLUSTER1_CONTEXT}" "${CLUSTER1_USER}" "${CLUSTER1_PASS}"
-install_istio "${CLUSTER1_NAME}" "${NETWORK1_ID}"
+install_istio "${CLUSTER1_NAME}" "${NETWORK1_ID}" "${MC_EAST_YAML}"
 
 echo "==== INSTALL ISTIO ON CLUSTER #2 [${CLUSTER2_NAME}] - ${CLUSTER2_CONTEXT}"
 switch_cluster "${CLUSTER2_CONTEXT}" "${CLUSTER2_USER}" "${CLUSTER2_PASS}"
-install_istio "${CLUSTER2_NAME}" "${NETWORK2_ID}"
+install_istio "${CLUSTER2_NAME}" "${NETWORK2_ID}" "${MC_WEST_YAML}"
+
+echo "==== DEPLOY ISTIO INGRESS GATEWAY ON CLUSTER #1 [${CLUSTER1_NAME}] - ${CLUSTER1_CONTEXT}"
+switch_cluster "${CLUSTER1_CONTEXT}" "${CLUSTER1_USER}" "${CLUSTER1_PASS}"
+kubectl apply -n istio-system -f "${SCRIPT_DIR}/../istio-gateway.yaml"
```

Comment:
Do we need the ingress gateway in each cluster? 

If so, you'll need to pass the context to each `kubectl apply`. e.g. `kubectl apply --context "${CLUSTER1_CONTEXT}" ...`.

If not, you can remove one of these `kubectl apply gateway`. I think it's used for traces to send traces from the "west" cluster to the "east" cluster but it may be used in other ways as well.

Reply:
> Do we need the ingress gateway in each cluster?

We need the gateway for cluster 1 only, so I removed the second occurrence.

> If so, you'll need to pass the context to each kubectl apply

Context of `kubectl` is switched by previous `switch_cluster` function